### PR TITLE
Feat/scaling - Finish development with simple scaling policy

### PR DESCRIPTION
**Pull request by mtulio**
Finish development. More information please have a look on README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,168 @@
...
-`cs_global_tags`: Global tags to apply on the resources
-`cs_lc_default`: Global options for LC module
-`cs_asg_default`: Global options for ASG
-`cloud_scaling_groups`: a list of scaling groups
+`cs_tags`: Global tags used on resources.
+`cs_ami_create`: Enable AMI creation (not supported, only false)
+`cs_lc_instance_type`: instance type to use to create Launch Configuration
+`cs_lc_spot_price`: When using spot, the Spot price
+`cs_lc_security_groups`: List of security groups
+`cs_asg_target_group_arms`: List of target group ARNs
```
Comment (raphapr) [inline README.md]: please fix typo `cs_asg_target_group_arms`

File: README.md [hunk 2]
```diff
@@ -1,39 +1,168 @@
...
+
+      ### Global CS LC
+      cs_lc_instance_type: c5.4xlarge
+      cs_lc_spot_price: 0.68
+      cs_lc_security_groups:
+        - apps
+        - api
+
+      ### Global CS ASG
+      cs_asg_target_group_arms:
```
Comment (raphapr) [inline README.md]: please fix typo `cs_asg_target_group_arms`

File: README.md [hunk 3]
```diff
@@ -1,39 +1,168 @@
...
+            instance_profile_name: instance-role-api
+            ebs_optimized: yes
+            image_id: ami-abcdef0123456789
+            spot_price: "{{ cs_lc_spot_price }}"
+          asg_spec:
+            min_size: 0
+            max_size: 5
+            desired_capacity: 0
+            health_check_type: "ELB"
+            target_group_arns: "{{ cs_asg_target_group_arms }}"
```
Comment (raphapr) [inline README.md]: please fix typo `cs_asg_target_group_arms`

File: tasks/aws-asg-policy/simple.yml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+- name: "aws | asg | {{ item_scale.asg_name }} | Update Alarms and link to Scaling Policies"
+  ec2_metric_alarm:
+    name: "{{ item.name }}"
+    state: present
+    metric: "CPUUtilization"
+    namespace: "AWS/EC2"
+    statistic: "Average"
+    comparison: "{{ item.comparison }}"
+    threshold: "{{ item.threshold }}"
+    period: 60
```
Comment (raphapr) [inline tasks/aws-asg-policy/simple.yml]: what about parameterize it?

Comment (mtulio) [reply tasks/aws-asg-policy/simple.yml]: Done

**Pull request merged**